src/service_account.rs

```diff
@@ -92,12 +92,12 @@ struct Claims<'a> {
     aud: &'a str,
     exp: i64,
     iat: i64,
-    subject: Option<&'a str>,
+    sub: Option<&'a str>,
```

Comment:
rather than changing the field would you mind adding `#[serde(rename = "sub")]` instead. I believe that's what I had intended to do but must have somehow forgot. I prefer the name subject in the code so that it doesn't conflict with the subtraction trait and remains the same name as the field in ServiceAccountFlow.

Reply:
Sure, no problem. That's another option I thought about. PR has been updated :)